src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+/* eslint-disable react/destructuring-assignment */
+
+/* eslint-disable no-useless-constructor */
+
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- Try fixing these errors. They are straightforward errors that can easily be fixed. Do the same for all components.